views/participant.html.twig

```diff
@@ -457,6 +596,57 @@ $(document).ready(function() {
         height: 50,
         displayValue: true
     });
+
+    var orgPatientStatusData = {{ orgPatientStatusData|json_encode|raw }};
+
+    // Switch to default tab if empty
+    if (!orgPatientStatusData) {
+        $('[href="#patientstatus"]').tab('show');
+    }
+
+    // Switch to patient status tab if there is a form error
+    if ($('.patient-status-form').find('div').hasClass('alert-danger')) {
+        $('[href="#patientstatus"]').tab('show');
+        // Display form
+        setTimeout(function () {
+            $('.btn-patient-status-update').trigger('click');
+        }, 100);
+    }
+
+    $('.patient-status-block').on('click', function () {
+        $('[href="#patientstatus"]').tab('show');
+    });
+
+    // Hide form by default if not empty
+    if (orgPatientStatusData) {
+        $('#patient-status-data-box').show();
+        $('#patient-status-form-box').hide();
+    }
+
+    $('.btn-patient-status-update').on('click', function () {
+        $('#patient-status-data-box').hide();
+        $('#patient-status-form-box').show();
+    });
+
+    $('.btn-patient-status-cancel').on('click', function () {
+        $('#patient-status-data-box').show();
+        $('#patient-status-form-box').hide();
+    });
+
+    // Display history in modal window
+    var psDetailsModal = $('#patient-status-details-modal');
+    $(".patient-status-details").on('click', function (e) {
+        e.preventDefault();
+        $(psDetailsModal).removeData('bs.modal');
+        $(psDetailsModal).modal({
+            remote: $(this).attr('data-href')
```

Comment:
The remote option [is deprecated](https://getbootstrap.com/docs/3.4/javascript/#modals-options).  We can leave this for now, but we should add a task to refactor.